ui/components/multichain/pages/review-permissions-page/review-permissions-page.tsx

```diff
@@ -0,0 +1,204 @@
+import { removePermissionsFor } from '../../../../store/actions';
+import {
+  AvatarFavicon,
+  AvatarFaviconSize,
+  Box,
+  Button,
+  ButtonIcon,
+  ButtonIconSize,
+  ButtonSize,
+  ButtonVariant,
+  Icon,
+  IconName,
+  IconSize,
+  Text,
+} from '../../../component-library';
+import { ToastContainer, Toast } from '../..';
+import { NoConnectionContent } from '../connections/components/no-connection';
+import { Content, Footer, Header, Page } from '../page';
+import { SubjectsType } from '../connections/components/connections.types';
+import { SiteCell } from '.';
+
+export const ReviewPermissions = () => {
+  const t = useI18nContext();
+  const dispatch = useDispatch();
+  const history = useHistory();
+  const urlParams: { origin: string } = useParams();
+  const securedOrigin = decodeURIComponent(urlParams.origin);
+  const [showAccountToast, setShowAccountToast] = useState(false);
+  const [showNetworkToast, setShowNetworkToast] = useState(false);
+  const activeTabOrigin: string = securedOrigin;
+  const subjectMetadata: { [key: string]: any } = useSelector(
+    getConnectedSitesList,
+  );
+  const connectedSubjectsMetadata = subjectMetadata[activeTabOrigin];
+  const connectedNetworks = useSelector((state) =>
+    getPermittedChainsForSelectedTab(state, activeTabOrigin),
+  );
+  const networksList = useSelector(getNonTestNetworks);
+  const connectedAccounts = useSelector((state) =>
+    getOrderedConnectedAccountsForConnectedDapp(state, activeTabOrigin),
+  );
+  const subjects = useSelector(getPermissionSubjects);
+  const grantedNetworks = networksList.filter(
+    (net: { chainId: any }) => connectedNetworks.indexOf(net.chainId) !== -1,
+  );
+  const hostName = getURLHost(securedOrigin);
+  const disconnectAllAccounts = () => {
+    const subject = (subjects as SubjectsType)[activeTabOrigin];
+
+    if (subject) {
+      const permissionMethodNames = Object.values(subject.permissions).map(
+        ({ parentCapability }: { parentCapability: string }) =>
+          parentCapability,
+      ) as string[];
+      if (permissionMethodNames.length > 0) {
+        const permissionsRecord: Record<string, string[]> = {
+          [activeTabOrigin]: permissionMethodNames,
+        };
+
+        dispatch(
+          removePermissionsFor(
+            permissionsRecord as Record<string, NonEmptyArray<string>>,
+          ),
+        );
+      }
+    }
+  };
+  return (
+    <Page
+      data-testid="connections-page"
+      className="main-container connections-page"
+    >
+      {connectedAccounts.length > 0 ? (
+        <>
+          <Header
+            backgroundColor={BackgroundColor.backgroundDefault}
+            startAccessory={
+              <ButtonIcon
+                ariaLabel={t('back')}
+                iconName={IconName.ArrowLeft}
+                className="connections-header__start-accessory"
+                size={ButtonIconSize.Sm}
+                onClick={() => (history as any).goBack()}
```

Comment:
Use of `any`

Reply:
I couldn't determine history here. So, it's fine to use any here